ZzaDashboard/ViewModel/MainViewModel.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows;
+using System.Windows.Input;
+using Zza.Data;
+
+namespace ZzaDashboard.ViewModel
+{
+    class MainViewModel : INotifyPropertyChanged
+    {
+        private Customer selectedCustomer;
+        public Customer SelectedCustomer
+        {
+            get => this.selectedCustomer;
+            set
+            {
+                if (value == this.selectedCustomer)
+                    return;
+
+                this.selectedCustomer = value;
+                this.OnPropertyChanged(nameof(this.SelectedCustomer));
+            }
+        }
+
+        private ObservableCollection<Customer> customers;
+        public ObservableCollection<Customer> Customers
+        {
+            get => this.customers;
+            set
+            {
+                if (value == this.customers)
+                    return;
+
+                this.customers = value;
+                this.OnPropertyChanged(nameof(this.Customers));
+            }
+        }
+
+        public event PropertyChangedEventHandler PropertyChanged;
+        protected void OnPropertyChanged(string propertyName)
+        {
+            if (this.PropertyChanged is null)
+                return;
+
+            this.PropertyChanged(this, new PropertyChangedEventArgs(propertyName));
+        }
+
+
+        public ICommand SaveCommand { get; set; }
+
+
+        public MainViewModel()
+        {
+            if (DesignerProperties.GetIsInDesignMode(new DependencyObject()))
+                return;
+
+            this.Customers = GetAllCustomers();
+
+            this.SaveCommand = new RelayCommand(SaveCommandExecuted, SaveCommandCanExecute);
+
+        }
+
+        private ObservableCollection<Customer> GetAllCustomers()
+        {
+            string hugeText = string.Empty;
+            ObservableCollection<Customer> customers = new ObservableCollection<Customer>();
```

Comment:
Lets use var in such cases.

Reply:
ok but just why? :-)

Reply:
The class is specified with constructor, so no need to waste such place in code for the same words.
I mean it is clear what type of the variable from initialization.